mention-bot.js

```diff
@@ -415,8 +420,8 @@ async function guessOwnersForPullRequest(
 ): Promise<Array<string>> {
   var diff = await fetch(repoURL + '/pull/' + id + '.diff');
   var files = parseDiff(diff);
-  var defaultOwners = getDefaultOwners(files, config.alwaysNotifyForPaths);
-
+  var defaultOwners = getMatchingOwners(files, config.alwaysNotifyForPaths);
+  var fallbackOwners = getMatchingOwners(files, config.fallbackNotifyForPaths);
   if (!config.findPotentialReviewers) {
       return defaultOwners;
```

Comment:
I get what you're saying, and that's definitely doable with `alwaysNotifyForPaths` - but is it the right way to do it?
Imho from a consistency perspective it's not, what I mean is that we currently support different flows (with `findPotentialReviewers` being enabled/disabled) and using these settings differently depending on the flow we're going with, might be kind of misleading to the end user.

The way I see this is:

> Find potential reviewers is enabled, .mention-bot uses blame history
> Always notify for paths is set, goes through that map of files/users and sees who needs to be additionally notified
> Fallback notify for paths is set, if no user found so far based on the two above, notify these set of users

or:

> Find potential reviewers is disabled, .mention-bot skips blame history
> Always notify for paths is set, goes through that map of files/users and sees who needs to be notified by default
> Fallback notify for paths is set, if no user found so far based on the previous one, notify these set of users

Meaning that based on the above, `fallbackNotifyForPaths` is used consistently independently of the flow we're going through instead of the current behaviour where it's only used when `findPotentialReviewers` is enabled forcing the user to set the "fallback notify for paths" mechanism in "always notify for paths" - right?

Let me know if I'm missing something.

Anyway, thanks for adding support for this, I definitely think it's useful :+1: